pkg/utils/utils.go

```diff
@@ -231,23 +230,25 @@ func GetAccount(connection *sdk.Connection, key string) (account *amv1.Account,
 	return
 }
 
-func ConfirmSend() error {
+func ConfirmSend() bool {
```

Comment:
I believe returning an error is better here. Everywhere this is being used, the `false` case is handled by printing and error and exiting anyway. Generally in Go the best practice is to bubble up the error to the calling logic and allowing the caller to log it and then make a decision based on it.

Reply:
In this case, with this dialog I tend to agree, as we exit if we don't `Continue y/N?`. Changed.